sections/advanced.md

```diff
@@ -182,6 +182,8 @@ Note that you need to be using at least git 2.24 for this hook.
 
 `pre-push` is triggered on `git push`.
 
+note: `pre-push` hooks are not triggered for ref deletions.
+
```

Comment:
there's a bit more nuance to this than just that -- the variables below imply this is already the case since there must be a from-ref and a to-ref

Reply:
Based on the environment variables, I was expecting `PRE_COMMIT_TO_REF` to be `0000000000000000000000000000000000000000` here.

Would you propose an alternate wording, or not documenting this at all?